x-pack/legacy/plugins/ml/server/models/data_recognizer/data_recognizer.js

```diff
@@ -839,12 +839,19 @@ export class DataRecognizer {
       }
     });
 
+    function processArrayValues(source, update) {
```

Comment:
unfortunately this solution will only check the first level of properties in the config object. e.g. `groups`
it won't catch deeper arrays like `influencers`
```
{
  "analysis_config": {
    "influencers": [
      "blah"
    ]
  }
}
```

Reply:
fixed in https://github.com/elastic/kibana/pull/49310/commits/f7813cce7fb597105e454215cc84d82b987236af